### PR TITLE
Remove .sh suffix from mainframer script filename.

### LINKED ISSUE #169: Rename mainframer.sh to mainframer.
by most used convention for shell scripts (ie google style guides for shell scripts) they should be named without file extension.

cc @ghostbuster91 @pawegio

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #169.

### CODE REVIEW

File: test/disabled_sample_buck.sh [hunk 1]
```diff
@@ -15,8 +15,8 @@ cp -a "$DIR/../samples/buck/." "$BUILD_DIR"
 # Overwrite config to work with test remote machine.
 setTestRemoteMachineInConfig
 
-# Run mainframer.sh that builds Buck project.
-bash "$REPO_DIR"/mainframer.sh 'buck clean && buck build sample'
+# Run mainframer that builds Buck project.
```
Comment (yunikkk) [inline test/disabled_sample_buck.sh:18]: Here and everywhere else - docs(samples/buck/README.md for this example) contains different form of script invocation.

**Review by yunikkk — APPROVED** [review]

**Review by arturdryomov — APPROVED** [review]

File: test/disabled_sample_buck.sh (hunk 1 above)
Comment (artem-zinnatullin) [reply test/disabled_sample_buck.sh:18]: You mean explicit invocation of `bash`?

Comment (yunikkk) [reply test/disabled_sample_buck.sh:18]: Yup

Comment (artem-zinnatullin) [reply test/disabled_sample_buck.sh:18]: Alright, I'll fix that

**Comment by artem-zinnatullin** [comment]
@yunikkk PTAL

**Review by yunikkk — APPROVED** [review]

**Pull request merged**